conf/airframes/tudelft/bebop_mav_course_exercise.xml

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop_avoider">
+  <description>Vision Course TUDelft V2019
```

Comment:
Since you were updating the years anyway ;)
```suggestion
  <description>MAVLAB Course TUDelft
```

Reply:
Nice catch :)